tests/driver/runner_test.py

```diff
@@ -2756,6 +2757,140 @@ async def test_create_snapshot_wait_for_completion(self, es):
             }
         })
 
+        params = {
```

Comment:
Unfortunately I cannot comment at the specific line but in `test_create_snapshot_no_wait` we still mock `es.snapshot.status` and assert later on that it is not called. IMHO we should remove this now because there is no chance we'd ever call it anymore in the new runner implementation.

Reply:
Addressed in 32ee4abf6a15d4158992c9843958b51e9a766cb5 + 2b242b2f6596c784674b908853e8ffbbdc83240e + 6523ec90ae29a2a70272865fdaa13d19c66c0bff